assets/blocks/single-course.scss

```diff
@@ -1,4 +1,3 @@
-
-@import 'take-course/take-course';
+@import 'button/style';
```

Comment:
@yscik I noticed in the https://github.com/Automattic/sensei/pull/3703 you moved some buttons styles from the button to the take-course and it conflicted with this PR. I undid it [here](https://github.com/Automattic/sensei/pull/3736/commits/ba8967dd7b1cefcfd4c4cf41cb97c1328e3c34ef) for this PR purpose. If you had some different thought when you changed that, please let me know. ;)

Reply:
> align buttons side by side

This doesn't seem to be suited for that. Here are a few buttons with left/right alignment:

<img width="1271" alt="image" src="https://user-images.githubusercontent.com/176949/97446788-20d63a80-192f-11eb-8762-d2f914e4e4b8.png">

Feels like the built-in block alignment controls work best for container blocks and for things like floating content outside the content margin. This change would make it impossible to actually align the buttons within the content. Users can use the Columns block to put blocks side by side (and we can too for the default template or in a block pattern)

I think the confusion for the alignment options was mainly because it was called `Align text left` at first, which was cleared up in #3703 along with the icons. So since text alignment isn't need for buttons and the block alignment works better with the existing control, this change and the related card might not be necessary.